### DISCUSSION TITLE
Output ":open" queries to script

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qutebrowser/qutebrowser#6803] [Q&A] [answered]
Is there a way to output the `:open` queries to be used in a script?

**Comment by The-Compiler** [comment]
I'm not sure I understand what you mean. Can you elaborate?

**Comment by LamprosPitsillos** [comment]
When i type `:open` a bunch of suggestions pop up like history, quick marks, bookmarks,  search engines etc.
I want to use those suggestions outside of qutebrowser , in a rofi/dmenu scripts lets say.
Is there a way to do that?

**Comment by The-Compiler** [comment, marked as answer]
You can process the data in the files manually, though the file format isn't guaranteed to stay stable. See [qutedmenu](https://github.com/qutebrowser/qutebrowser/blob/master/misc/userscripts/qutedmenu) and [dmenu_qutebrowser](https://github.com/qutebrowser/qutebrowser/blob/master/misc/userscripts/dmenu_qutebrowser) for some inspiration.
